tardis/montecarlo/montecarlo_numba/r_packet.py

```diff
@@ -621,4 +621,4 @@ def angle_aberration_LF_to_CMF(r_packet, time_explosion, mu):
 def test_for_close_line(r_packet, line_id, nu_line, numba_plasma):
     r_packet.is_close_line = abs(
         numba_plasma.line_list_nu[line_id] - nu_line
-    ) < (nu_line * CLOSE_LINE_THRESHOLD)
+    ) < (nu_line * CLOSE_LINE_THRESHOLD)
```

Comment:
Remove this change

Reply:
The same thing happens above, I don't remember making any change, and when I try to edit the file it recognizes it as the same code, so I cannot commit.